daemon/cmd/app.go

```diff
@@ -0,0 +1,42 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Authors of Cilium
+
+package cmd
+
+import (
+	"context"
+
+	"go.uber.org/fx"
+)
+
+func runApp() {
+	ctx, cancel := context.WithCancel(context.Background())
+	app := fx.New(
+		fx.WithLogger(newAppLogger),
+		fx.Supply(fx.Annotate(ctx, fx.As(new(context.Context)))),
+		cleanerModule,
+		fx.Provide(daemonModule),
+		fx.Invoke(func(*Daemon) {}),
```

Comment:
Do we need this line?

Reply:
Yes, for now as nothing depends on daemon and we need an invoke to pull on the constructors. This will go away with the next PR.